### ISSUE TITLE
LogExpert stopped working due "NotImplementedException" after playing with AutoColumizer

### ISSUE THREAD

**Issue by hoppfrosch** [zarunbal/LogExpert#284]
# Bug

After playing around with Auto-Columizer, I got in a situation where LogExpert stopped working: Each time I start Logexpert (with or without logfile), the GUI starts up with the columns "Line" and "Text" entirely black (see https://i.imgur.com/XEC9hGj.png). Trying to select anything from the menu/Logexpert-GUI results in an error box, saying:

-----

`Die Methode oder der Vorgang ist nicht implementiert.
NotImplementedException
   bei LogExpert.AutoColumnizer.GetTimestamp(ILogLineColumnizerCallback callback, ILogLine line)
   bei LogExpert.Controls.LogWindow.LogWindow.GetTimestampForLineForward(Int32& lineNum, Boolean roundToSeconds)
   bei LogExpert.Controls.LogWindow.LogWindow.SetTimestampLimits()
   bei LogExpert.Controls.LogWindow.LogWindow.LogWindowActivated()
   bei LogExpert.Controls.LogTabWindow.LogTabWindow.OnDockPanelActiveContentChanged(Object sender, EventArgs e)
   bei WeifenLuo.WinFormsUI.Docking.DockPanel.OnActiveContentChanged(EventArgs e)
   bei WeifenLuo.WinFormsUI.Docking.DockPanel.FocusManagerImpl.RefreshActiveWindow()
   bei WeifenLuo.WinFormsUI.Docking.DockPanel.FocusManagerImpl.HookEventHandler(Object sender, HookEventArgs e)
   bei WeifenLuo.WinFormsUI.Docking.DockPanel.FocusManagerImpl.LocalWindowsHook.OnHookInvoked(HookEventArgs e)
   bei WeifenLuo.WinFormsUI.Docking.DockPanel.FocusManagerImpl.LocalWindowsHook.CoreHookProc(Int32 code, IntPtr wParam, IntPtr lParam)`

----

After this, logexpert dies. Therefore I cannot usr logexpert anymore. 

Not sure how to reproduce this error, as I am not able to experiment with logexpert anymore, as it doesn't work anymore on my machine. What I remember what I did: I played around with Auto-Columizer and I tried to define a Setting to use Auto-Columizer for a given filepattern ...

I reinstalled logexpert, after this I updated logexpert (from 1.9.0 to 1.9.1) - but without success: the erroneous behaviour consisted.

**Any way to "reset" logexpert to default behaviour?**

Important questions
- LogExpert Version: 1.9.0 / 1.9.1, installed via chocolatey
- Windows Version: Win10 Professional
- Lokal file or Network: error occurs on startup without any file
- Used Columnizer: AutoColumizer
- Is this version specific or a new problem: don't know

**Comment by hoppfrosch** [comment]
Solved problem - my logexpert was configured to always open last opened logfile. This logfile matched the file pattern, which was configured to use AutoColumnizer.

Solution: Opened logexpert with logfile which doesn't match filepattern. This works without any problem. After having logexpert running again, I deleted the "Filepattern with Autocolumnizer" in my Settings .... and everything worked fine again. 

The described problem is a problem with configuring file patterns to use AutoColumnizer ....
